Luke/common/Status.py

```diff
@@ -0,0 +1,2 @@
+class Status:
+    nothing, matched, reboot, done = range(4)
```

Comment:
lets say we get a changestatus request with a number which is not defined here.
what would happen?
I'm asking because I haven't figured out where we translate from number to status.

another point: does it make more sense to define Status in a separate file in common, or in the same file as BareMetal? I don't have a strong opinion here